Reply/app/src/main/java/com/example/reply/data/local/LocalEmailsDataProvider.kt

```diff
@@ -0,0 +1,333 @@
+object LocalEmailsDataProvider {
+
+    private val threads = listOf(
+        Email(
+            8L,
+            LocalAccountsDataProvider.getContactAccountByUid(13L),
+            listOf(LocalAccountsDataProvider.getDefaultUserAccount()),
+            "Your update on Google Play Store is live!",
+            """
+              Your update, 0.1.1, is now live on the Play Store and available for your alpha users to start testing.
+              
+              Your alpha testers will be automatically notified. If you'd rather send them a link directly, go to your Google Play Console and follow the instructions for obtaining an open alpha testing link.
+            """.trimIndent(),
+            mailbox = MailboxType.TRASH,
+            createAt = "3 hours ago",
+        ),
+        Email(
+            5L,
+            LocalAccountsDataProvider.getContactAccountByUid(13L),
+            listOf(LocalAccountsDataProvider.getDefaultUserAccount()),
+            "Update to Your Itinerary",
+            "",
+            createAt = "2 hours ago",
+        ),
+        Email(
+            6L,
+            LocalAccountsDataProvider.getContactAccountByUid(10L),
+            listOf(LocalAccountsDataProvider.getDefaultUserAccount()),
+            "Recipe to try",
+            "Raspberry Pie: We should make this pie recipe tonight! The filling is " +
+                "very quick to put together.",
+            createAt = "2 hours ago",
+            mailbox = MailboxType.SENT
+        ),
+        Email(
+            7L,
+            LocalAccountsDataProvider.getContactAccountByUid(9L),
+            listOf(LocalAccountsDataProvider.getDefaultUserAccount()),
+            "Delivered",
+            "Your shoes should be waiting for you at home!",
+            createAt = "2 hours ago",
+        ),
+        Email(
+            9L,
+            LocalAccountsDataProvider.getContactAccountByUid(10L),
+            listOf(LocalAccountsDataProvider.getDefaultUserAccount()),
+            "(No subject)",
+            """
+            Hey, 
+            
+            Wanted to email and see what you thought of
+            """.trimIndent(),
+            createAt = "3 hours ago",
+            mailbox = MailboxType.DRAFTS
+        ),
+        Email(
+            1L,
+            LocalAccountsDataProvider.getContactAccountByUid(6L),
+            listOf(LocalAccountsDataProvider.getDefaultUserAccount()),
+            "Brunch this weekend?",
+            """
+                I'll be in your neighborhood doing errands and was hoping to catch you for a coffee this Saturday. If you don't have anything scheduled, it would be great to see you! It feels like its been forever.
+
+                If we do get a chance to get together, remind me to tell you about Kim. She stopped over at the house to say hey to the kids and told me all about her trip to Mexico.
+
+                Talk to you soon,
+
+                Ali
+            """.trimIndent(),
+            createAt = "40 mins ago",
```

Comment:
[suggestion] could be nice to have this as actual dates and then compute these strings instead, but I understand its a sample :)